crate_universe/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub mod cli;
 
 mod config;
 mod context;
-mod lockfile;
+pub mod lockfile;
```

Comment:
Maybe we should have an `api` module instead? I worry about leaking things from `lockfile` since now we'd need to explicitly state `pub(crate)` Maybe that's a good thing though? I'll defer to you 😄